### ISSUE TITLE
Feature request license

### ISSUE THREAD

**Issue by emmaladolphi** [switchonproject/cids-custom-switchon#130]
When using the metadata wizard, step 5 (license):
Limitations on Public Access; it follows the Inspire standard which is good, but to have no limitation as default can be problematic. If we don't know the limitation it doesn't feel safe to add no limitation since this could perhaps lead to legal problems both for the user and us. Even if the option 'unknown' doesn't exist in the Inspire standard we think it would be best to add this option. This way we can add datasets where the limitations are not known and we avoid legal trouble.

![limitations on public access](https://cloud.githubusercontent.com/assets/8414617/8427226/24e56d8a-1f16-11e5-9ccb-65680d7b444c.jpg)

In regards to the Conditions for Access and Use: the options in the scroll bar is confusing. A user may not know the terminology which makes it very difficult to add and option. It would be helpful to use easier terminology or have a info button which leads to a pop-up window or something with definition/explanation of the licenses (choices). 

Often the licenses of the data contain many criteria (homemade licenses) so to only have one choice in the scroll bar isn't enough. It would be good if one could add multiple licenses (maybe have a similar function as for the Cuahsi keywords). 

To summarize: legally 'no limitation' is not a useful default. The conditions for access and use are very confusing and difficult to work with since we need to be able to add multiple and add some sort of description/definition to make the tool easier to use.

**Comment by p-a-s-c-a-l** [comment]
- If you select any of the licenses, Conditions for Access, limitations, etc. a description is shown in the 'Completion Guide' Box
- adding multiple licenses would be imho more confusing for the user! Furthermore, if the user is allowed to define arbitrary licenses we'll have the same issues as with the free keywords. Solution: select 'other' as license if none of the selections matches the homemade license.
- I'll add the option 'unknown' to the limitations on public access.

**Comment by esafa** [comment]
I found the problem (and a potential solution).. Sorry for long post.

Inspire legislation says *government authorities *must release information
unless the disclosure would adversely affect "the course of justice",
intellectual
property rights etc...
So "limitation on public access" is for goverment authorities, not research
institutes etc.  The options in the inspire standard are regulated by
the Inspire
directive of 2007.. there are only 8 allowed reason to refuse access. If
none of them applies there is "no limitation". This works for government
authorities but not for e.g. privately owned research institutes... they
are not bound by Inspire regulations so they may have "other reasons" to
limit public access but the inspire standard do not allow any other
"reasons" to limit access.

So... Inspire was designed for goverments and we are using it for something
else.

As background, please read the legislation and the guideline doc.

1.   By way of derogation from Article 11(1), Member States may limit
public access to spatial data sets and services through the services
referred to in point (a) of Article 11(1) where such access would adversely
affect international relations, public security or national defence.

By way of derogation from Article 11(1), Member States may limit
public access to spatial data sets and services through the services
referred to in points (b) to (e) of Article 11(1), or to the e-commerce
services referred to in Article 14(3), where such access would adversely
affect any of the following:

(a)

the confidentiality of the proceedings of public authorities, where such
confidentiality is provided for by law;

(b)

international relations, public security or national defence;

(c)

the course of justice, the ability of any person to receive a fair trial or
the ability of a public authority to conduct an enquiry of a criminal or
disciplinary nature;

(d)

the confidentiality of commercial or industrial information, where such
confidentiality is provided for by national or Community law to protect a
legitimate economic interest, including the public interest in maintaining
statistical confidentiality and tax secrecy;

(e)

intellectual property rights;

(f)

the confidentiality of personal data and/or files relating to a natural
person where that person has not consented to the disclosure of the
information to the public, where such confidentiality is provided for by
national or Community law;

(g)

the interests or protection of any person who supplied the information
requested on a voluntary basis without being under, or capable of being put
under, a legal obligation to do so, unless that person has consented to the
release of the information concerned;

(h)

the protection of the environment to which such information relates, such
as the location of rare species.

2.   The grounds for limiting access, as provided for in paragraph 1, shall
be interpreted in a restrictive way, taking into account for the particular
case the public interest served by providing this access. In every
particular case, the public interest served by disclosure shall be weighed
against the interest served by limiting or conditioning the access. Member
States may not, by virtue of points (a), (d), (f), (g) and (h) of
paragraph 1, limit access to information on emissions into the environment.

3.   Within this framework, and for the purposes of the application of
point (f) of paragraph 1, Member States shall ensure that the requirements
of Directive 95/46/EC are complied with.


The actual Inspire guidelines on this is at page 51 in
http://inspire.ec.europa.eu/documents/Metadata/MD_IR_and_ISO_20131029.pdf


To be conformant with inspire we need to do the following:

*TG Requirement 32: Limitations on public access shall be represented by at
least one of these metadata elements: *

*MD_LegalConstraints. accessConstraints *

*MD_LegalConstraints. otherConstraints *


*MD_SecurityConstraints. classification*


So I see a way out. As the guideline points out there may be three
scenarios according to the tables above: • There may be no limitation on
public access; • There may be only a classification property when
expressing a security constraint (i.e., this is an instance of
MD_SecurityConstraints or one of its subclasses); • There may be one or
more instances of the accessConstraints property, possibly associated with
one or more instances of otherRestrictions property (i.e, this is an
instance of MD_LegalConstraints);


So to *avoid *getting into legal issues by lying about "no limitation" we
can instead *only *use *MD_LegalConstraints. accessConstraints. *The value
has to be according to the Codelist defined in B.5.24 of ISO 19115... and
here I believe we can use *"otherRestrictions (limitation not listed)."*


There is a big difference is saying "no limitation" and "limitation not
listed".


Problem solved?




On Tue, Jun 30, 2015 at 10:53 AM, emmaladolphi <notifications@github.com>
wrote:

> When using the metadata wizard, step 5 (license):
> Limitations on Public Access; it follows the Inspire standard which is
> good, but to have no limitation as default can be problematic. If we don't
> know the limitation it doesn't feel safe to add no limitation since this
> could perhaps lead to legal problems both for the user and us. Even if the
> option 'unknown' doesn't exist in the Inspire standard we think it would be
> best to add this option. This way we can add datasets where the limitations
> are not known and we avoid legal trouble.
>
> [image: limitations on public access]
> <https://cloud.githubusercontent.com/assets/8414617/8427226/24e56d8a-1f16-11e5-9ccb-65680d7b444c.jpg>
>
> In regards to the Conditions for Access and Use: the options in the scroll
> bar is confusing. A user may not know the terminology which makes it very
> difficult to add and option. It would be helpful to use easier terminology
> or have a info button which leads to a pop-up window or something with
> definition/explanation of the licenses (choices).
>
> Often the licenses of the data contain many criteria (homemade licenses)
> so to only have one choice in the scroll bar isn't enough. It would be good
> if one could add multiple licenses (maybe have a similar function as for
> the Cuahsi keywords).
>
> To summarize: legally 'no limitation' is not a useful default. The
> conditions for access and use are very confusing and difficult to work with
> since we need to be able to add multiple and add some sort of
> description/definition to make the tool easier to use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/switchonproject/cids-custom-switchon/issues/130>.
>



-- 
Esa Falkenroth
mobile: 0702-104028 (intl +46702104028)
work: 011-4958298 (intl +46114958298)
skype: Esa.Falkenroth
email: nospam@falkenroth.se (for business mail)
email: esa@falkenroth.se (for private mail)

**Comment by p-a-s-c-a-l** [comment]
Thanks for the explanation. I'll add "limitation not listed" to the selection box and change the limitations of all resources that have been set by default to "no limitation" to "limitation not listed".

**Comment by p-a-s-c-a-l** [comment]
done
